src/customize-pop/dock-settings.md

```diff
@@ -0,0 +1,103 @@
+# Dock Settings
+
+This section describes settings located in the `Dock` tab for modifying Dock visibility behavior and Dock icons.
+
+![Customize Dock](/images/customize-desktop/customize-dock.png)
+
+## Enable Dock
+
+Disable or enable the Dock's visibility.
```

Comment:
Since there is a later section all about visibility, it might be simpler if you just say `Disable or enable the Dock` here.